### ISSUE TITLE
Bar / Curve display

### ISSUE THREAD

**Issue by davidrohr** [kotelnik/plasma-applet-thermal-monitor#25]
It would be very cool if the widget would support some more graphical output. A bar display could save some space, while a curve showing the temperature of the last minutes could give a better overview.

**Comment by kotelnik** [comment]
Agreed, I'll do that (similarly like it is in resources-monitor, or network-monitor).
